File: Sample/src/net/nend/sample/ListViewActivity.java
package net.nend.sample;

import java.util.ArrayList;

import net.nend.android.NendAdView;
import android.app.ListActivity;
import android.os.Bundle;
import android.widget.ArrayAdapter;
import android.widget.ListView;

/**
 * ListViewに表示するサンプル
 */
public class ListViewActivity extends ListActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        
        ListView listView = getListView();
        NendAdView adView = new NendAdView(this, 3174, "c5cb8bc474345961c6e7a9778c947957ed8e1e4f");
        adView.loadAd();
        listView.addHeaderView(adView);
        
        ArrayList<String> list = new ArrayList<String>();
        for (int i = 0; i < 100; i++) {
            list.add("item"+i);
        }
        
        ArrayAdapter<String> adapter = new ArrayAdapter<String>(this,
                android.R.layout.simple_list_item_1, list);
        setListAdapter(adapter);
    }
}

File: Sample/kotlin/src/main/java/net/nend/sample/kotlin/banner/ListViewActivity.kt
package net.nend.sample.kotlin.banner

import android.app.ListActivity
import android.os.Bundle
import android.widget.ArrayAdapter
import net.nend.android.NendAdView
import net.nend.sample.kotlin.banner.BannerSampleActivity.Companion.BANNER_API_KEY_320_50
import net.nend.sample.kotlin.banner.BannerSampleActivity.Companion.BANNER_SPOT_ID_320_50

/**
 * ListViewに表示するサンプル
 */
class ListViewActivity : ListActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        NendAdView(this, BANNER_SPOT_ID_320_50, BANNER_API_KEY_320_50).run {
            listView.addHeaderView(this)
            loadAd()
        }

        val list = (0..99).map { i -> "item$i" }
        listAdapter = ArrayAdapter(this, android.R.layout.simple_list_item_1, list)
    }
}